p2p/net/conn/dial.go

```diff
@@ -48,11 +53,15 @@ func (d *Dialer) Dial(ctx context.Context, raddr ma.Multiaddr, remote peer.ID) (
 
 		if d.PrivateKey == nil {
 			log.Warning("dialer %s dialing INSECURELY %s at %s!", d, remote, raddr)
+			log.Event(ctx, "connDialInsecure", logdial)
```

Comment:
Thought: rather than producing another event, `d.PrivateKey == nil` is known apriori. Let `insecure` be included on the original event.